vividus-extension-selenium/src/main/java/org/vividus/bdd/steps/WebDriverSetupSteps.java

```diff
@@ -26,15 +29,18 @@
 import org.vividus.selenium.IWebDriverProvider;
 import org.vividus.selenium.manager.IWebDriverManagerContext;
 
-public class VividusWebDriverSetupSteps
+public class WebDriverSetupSteps
 {
+    private final WebDriverSessionScope webDriverSessionScope;
     private final IWebDriverProvider webDriverProvider;
     private final IWebDriverManagerContext webDriverManagerContext;
     private final IBddRunContext bddRunContext;
 
-    public VividusWebDriverSetupSteps(IWebDriverProvider webDriverProvider,
+    public WebDriverSetupSteps(WebDriverSessionScope webDriverSessionScope, IWebDriverProvider webDriverProvider,
             IWebDriverManagerContext webDriverManagerContext, IBddRunContext bddRunContext)
     {
+        isTrue(webDriverSessionScope != null, "Application session scope is not set");
```

Comment:
Hmm what should I do if I get this error and I'm not aware what the "Application session scope" is? :)

Reply:
RTFM :)